deploy/cephcsi/image/Dockerfile

```diff
@@ -3,7 +3,7 @@ LABEL maintainers="Ceph-CSI Authors"
 LABEL description="Ceph-CSI Plugin"
 
 # Removing ceph-iscsi repo to workaround the repo issue while upgrading
-RUN rm -f /etc/yum.repos.d/ceph-iscsi.repo && yum -y update && yum clean all
+#RUN rm -f /etc/yum.repos.d/ceph-iscsi.repo && yum -y update && yum clean all
```

Comment:
Maybe you want to use the Dockerfile from https://github.com/nixpanic/ceph-csi/commit/a07b3603cf30142d7690cbcd2fab0cac467abbbb ? You'll need to add `--enablerepo=centos-ceph-nautilus-test` while installing `ceph-base` though.

That also addresses #573 and improves things a bit.

Reply:
I will leave this as it is, more info why it was added https://github.com/ceph/ceph-csi/pull/273#discussion_r269926684